opencog/atoms/base/atom_types.script

```diff
@@ -338,8 +338,8 @@ DUAL_LINK <- SATISFYING_LINK
 // Should these inherit from EvaluatableLink ??
 // Actually, these probably should inherit from PrenexLink, because
 // we typically want to maintain these in prenex normal form.
```

Comment:
This comment should be given some thought. But I'm not sure. PrenexLink provides assorted methods that are handy for rewriting links to make sure that they stay in prenex form, but its not at all clear if or how the ExistsLink could make use of that.   Its not urgent, we can defer this decision until later (and preserve this comment, for now)